client/src/components/elements/TextBox/TextArea/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { ChangeEvent, TextareaHTMLAttributes, useState } from 'react'
+import styles from './textarea.module.scss'
+
+interface TextareaProps extends TextareaHTMLAttributes<HTMLTextAreaElement> {
+  label: string
+}
+
+interface TextareaState {
+  textValue: string
+}
+
+// eslint-disable-next-line react/prop-types
+const Textarea: React.FC<TextareaProps> = ({ label, ...props }) => {
+  const [state, setState] = useState<TextareaState>({
+    textValue: '',
+  })
+
+  const handleTextareaChange = (event: ChangeEvent<HTMLTextAreaElement>) => {
+    setState((prev) => ({
+      ...prev,
+      textValue: event.target.value,
+    }))
+  }
+
+  return (
+    <div className={styles.container}>
+      <label className={styles.Label} htmlFor="textarea">
```

Comment:
your use of html for should be dynamic

you can tie it with the id, but make sure If an id is present, it becomes a priority